src/main.js

```diff
@@ -26,30 +25,12 @@ let load = base.loader({
     setup: ({profile}) => base.config({profile}),
   },
 
-  influx: {
-    requires: ['cfg'],
-    setup: ({cfg}) => {
-      if (cfg.influx.connectionString) {
-        return new base.stats.Influx(cfg.influx);
-      }
-      return new base.stats.NullDrain();
-    }
-  },
-  raven: {
-    requires: ['cfg'],
-    setup: ({cfg}) => {
-      if (cfg.raven.sentryDSN) {
-        return new raven.Client(cfg.raven.sentryDSN);
-      }
-      return null;
-    }
-  },
   monitor: {
-    requires: ['cfg', 'influx', 'process'],
-    setup: ({cfg, influx, process}) => base.stats.startProcessUsageReporting({
-      drain:      influx,
-      component:  cfg.app.statsComponent,
-      process:    process
+    requires: ['profile', 'cfg'],
+    setup: ({profile, cfg}) => base.monitor({
+      project: 'taskcluster-queue',
```

Comment:
qq, are we using this scheme elsewhere too? Just thinking `taskcluster-` might be a bit long.

Otherwise, nvm...